Source/Core/Common/ENetUtil.cpp

```diff
@@ -39,6 +39,12 @@ int ENET_CALLBACK InterceptCallback(ENetHost* host, ENetEvent* event)
 
 bool SendPacket(ENetPeer* socket, const sf::Packet& packet, u8 channel_id)
 {
+  if (!socket)
+  {
+    ERROR_LOG_FMT(NETPLAY, "No socket to send to given.");
```

Comment:
```suggestion
    ERROR_LOG_FMT(NETPLAY, "No socket to send to, given.");
```

Reply:
This suggestion makes no sense.

Reply:
I disagree, without the comma, it seems like you are trying to send a socket to given.

Reply:
I see what you mean but that's not how English grammar works. There's probably a better way to phrase this that's clearer.